crates/common/src/commit/client.rs

```diff
@@ -69,7 +70,7 @@ impl SignerClient {
     pub async fn request_signature(
```

Comment:
Split this client SDK method, too, in order to provide a couple of well-typed methods (with proper return types) instead of one, which is handling opaque byte vectors.

Reply:
Resolved in [eba97b0](https://github.com/Commit-Boost/commit-boost-client/pull/87/commits/eba97b0ca03b140bbfdc9f4b7b8ad4adf382776b)